nuxt.config.js

```diff
@@ -42,7 +42,6 @@ export default {
     meta: [
       { charset: 'utf-8' },
       { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-      { hid: 'description', name: 'description', content: `${config.appTitle}, multi, multi.coop, multi-site-app` },
```

Comment:
current change doesn't seem to have solved the issue, now there is no meta description displayed :sweat_smile: (PR 44, 43 is another PR to compare)
![image](https://github.com/multi-coop/multi-site-app/assets/7147385/b78953d0-023a-46cc-9b34-d3676ce754e8)